docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -0,0 +1,520 @@
+  4. [Reusing or sharing RCU tables](#reusing-or-sharing-rcu-tables).
+
+
+
+##### Set up and initialize an RCU database
+
+A JRF domain requires an infrastructure database and also requires initializing this database with a schema and a set of tables. The following example shows how to set up a sample RCU database and use the RCU tool to create the infrastructure schema for a JRF domain. The RCU database is set up with the following attributes:
+
+| Attribute | Value |
+| --------- | ----- |
+| database Kubernetes namespace | `default` |
+| database Kubernetes pod | `oracle-db` |
+| database image | `container-registry.oracle.com/database/enterprise:12.2.0.1-slim` |
+| database password | `Oradoc_db1` |
+| infrastructure schema prefix | `FMW1` |
+| infrastructure schema password | `Oradoc_db1` |
+| database URL | `oracle-db.default.svc.cluster.local:1521/devpdb.k8s` |
+
+
+1. Ensure you have access to the database image, and then deploy it:
+
+   - Use a browser to log in to `https://container-registry.oracle.com`, select `database->enterprise` and accept the license agreement.
+
+   - Get the database image:
+     - In the local shell, `docker login container-registry.oracle.com`.
+     - In the local shell, `docker pull container-registry.oracle.com/database/enterprise:12.2.0.1-slim`.
+
+
+       {{% notice note %}} If a local Docker login and manual pull of `container-registry.oracle.com/database/enterprise:12.2.0.1-slim` is not sufficient (for example, if you are using a remote Kubernetes cluster), then uncomment the `imagePullSecrets` stanza in `$WORKDIR/k8s-db-slim.yaml` and create the image pull secret as follows:
+
+              ```
+              kubectl create secret docker-registry regsecret \
+                --docker-server=container-registry.oracle.com \
+                --docker-username=your.email@some.com \
+                --docker-password=your-password \
+                --docker-email=your.email@some.com
+              ```
+       {{% /notice %}}
+
+
+   - Use the sample script in `$SRCDIR/kubernetes/samples/scripts/create-oracle-db-service` to create an Oracle database running in the pod, `oracle-db`.
+
+      **NOTE**: If your database image access requires the `regsecret` image pull secret that you optionally created above, then pass `-s regsecret` to the `start-db-service.sh` command line.
+
+     ```
+     cd $SRCDIR/kubernetes/samples/scripts/create-oracle-db-service
+     start-db-service.sh
+     ```
+
+     This script will deploy a database with the URL, `oracle-db.default.svc.cluster.local:1521/devpdb.k8s`, and administration password, `Oradoc_db1`.
+
+     {{% notice warning %}} The Oracle database Docker images are supported only for non-production use. For more details, see My Oracle Support note: Oracle Support for Database Running on Docker (Doc ID 2216342.1) : All the data is gone when the database is restarted.
```

Comment:
"Oracle Database Docker images".... capitalize Database.

Reply:
Yes, I missed that.

Reply:
Ok.

Reply:
Fixed.